build.go

```diff
@@ -26,9 +29,16 @@ func Build(
 			return packit.BuildResult{}, err
 		}
 
+		var layers []packit.Layer
+		if _, err := os.Stat(modCacheLayer.Path); err == nil {
```

Comment:
I believe that `fs.IsEmptyDir()` both checks if the files directory exists and it has any contents. So you might not need the `os.Stat()` call

Reply:
Upon a closer reading of the `fs.IsEmptyDir()` it will return false if the directory does not exist, so I retract my former statement.